blueprints/ember-nrg-ui/index.js

```diff
@@ -42,6 +42,12 @@ module.exports = {
   afterInstall() {
     const blueprint = this;
     const nodePackages = [
+      {
+        name: 'sass',
+        target: '1.43.5',
+      },
+    ];
+    const AddOns = [
```

Comment:
This should be camelCased.

```suggestion
    const addOns = [
```